javascript/testing_javascript/more_testing.md

```diff
@@ -65,7 +65,7 @@ If we had written this program with TDD it is very likely that it would have loo
 ### Assignment 
 
 <div class="lesson-content__panel" markdown="1">
-1. If you haven't already, watch the 'mocking' videos from [this series](https://www.youtube.com/watch?v=3PjdxjWK0F0).
+1. If you haven't already, watch the 'mocking' videos from [this series](https://www.youtube.com/watch?v=3PjdxjWK0F0). The concept of dependency injection is mentioned quite a bit in this video, so if you are unfamiliar with the topic, you might want to read [this article](https://javascript.plainenglish.io/dependency-injection-in-javascript-539f7f6cfbee) first. If you prefer a video alternative, you can watch [this video](https://youtu.be/0X1Ns2NRfks?si=JrsXRvoe4FIHcOf0) until [03:50](https://youtu.be/0X1Ns2NRfks?si=-yQu7SvuFApoVhXw&t=230).
```

Comment:
@ManonLef was your intent in the original issue to include both an article and video here? Wondering if it makes more sense to include the video as an additional resource if it covers a bit more than the article does.

Reply:
That is the video that should only be watched until 03:50. The assignment step already mentions this.

I understand that people could easily go down a rabbit hole if they miss the timestamp or simply decide to watch the whole video out of fear of missing out. I am not opposed to omitting the video entirely or including it in the additional resources with a timestamp.

Reply:
I'm good with including it if others feel it makes sense. I think I just partially lean more towards favoring articles over videos when we can, especially in this case where it's more of a "or watch the first few minutes of this video" rather than what feels like a "required" assignment item.

Either way, I'd say we should explicitly make them their own items rather than tacked on. So something more along the lines of:

```suggestion
1. Read about [dependency injection in JavaScript](https://javascript.plainenglish.io/dependency-injection-in-javascript-539f7f6cfbee), as it is a concept mentioned quite a bit in the videos from the next assignment item.
1. Watch the [mocking videos by Fun Fun Function](https://www.youtube.com/watch?v=3PjdxjWK0F0). You only need to watch the linked video as well as the next two videos in his unit testing series ("mocking continued" and "red, green, refactor, and Jest mocks").
```

If we do want to include the video on injection still, then it could be added to the new assignment 1:

```
1. Read about [dependency injection in JavaScript](https://javascript.plainenglish.io/dependency-injection-in-javascript-539f7f6cfbee), as it is a concept mentioned quite a bit in the next assignment item. If you prefer a video alternative, you can watch [dependency injection basics by Fun Fun Function](https://youtu.be/0X1Ns2NRfks?si=JrsXRvoe4FIHcOf0), up to the 3:50 timestamp only.
```

(we shouldn't link to the video again for the 3:50 time stamp)

Reply:
I see you requested my review @thatblindgeye and I agree with the suggestions you added above. I noticed the other PR is a bit stale, are you still waiting on changes for it?

Reply:
Not currently I don't believe. At this point I'm holding off on it since we'll need to decide whether to keep the Fun Fun Function videos that are in this PR, or replace them with the resource that's in the other PR (which may then make this PR no longer necessary)

Reply:
Taking another look at this after some time, and there's 2 ways I think we could go about resolving this issue:

- First is we keep the Fun Fun Function videos on mocking. If we do, we need to remove the dependency injection video from him as well, as the second assignment item is already totally over an hour in content that a user *has* to go through.
- The second is we replace the Fun Fun Function link for mocking with either the video that was attempted to be introduced in #26870 , or another video/resource. Considering we're asking users to sit through over an hour of video content, I'm personally in favor of this option.

cc @CCeph @ManonLef

Reply:
I agree with you @thatblindgeye , the second option might be best. I am a tad attached to Fun Fun Functions because I think it's still very comprehensive. But it might be getting a bit outdated. 

I have not gone through the whole other video (collection?) from the other resource, have you? I can't vouch for its quality. 

Also are you still willing to work on this @CCeph ?

Reply:
Any update on this? #28379 regards the Mocking video by FFF, so if we're planning to replace it with something else, then that'd also resolve the linked issue.

If needed, we can close this PR due to inactivity and open a new issue for someone to be assigned to.

Reply:
I'd say replace the FFF video(s) for this lesson, two birds one stone and all. @MaoShizhong

Reply:
@thatblindgeye Just to confirm, the replacement link would be https://www.youtube.com/watch?v=ajiAl5UNzBU&t=3024s ?

Reply:
I believe so, yes. Been a minute since I looked at this or the 26870 PR, but that sounds accurate (unless anyone else has another suggestion for an alternative of course)